changelog/unreleased/kong/feat-pdk-unlimited-body-size.yml

```diff
@@ -0,0 +1,3 @@
+message: Added negative value to support unlimited body size.
```

Comment:
Should we explain it in more details. Such as it still limited by nginx's max_body_size directive.

Reply:
resolved